src/Property/PropertyValueExtractor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\ConverterBundle\Property;
+
+use Neusta\ConverterBundle\Exception\PropertyException;
+
+class PropertyValueExtractor
+{
+    /**
+     * @throws PropertyException
+     */
+    public static function extractValue(object $source, string $propertyName): mixed
+    {
+        try {
+            foreach (['get', 'is', 'has'] as $prefix) {
+                if (method_exists($source, $prefix . ucfirst($propertyName))) {
+                    return $source->{$prefix . ucfirst($propertyName)}();
+                }
+            }
+        } catch (\Throwable $exception) {
+            throw new PropertyException($propertyName, $exception->getMessage(), $exception->getCode(), $exception);
+        }
+        return null;
```

Comment:
I think we should throw an exception instead of returning `null` if we're not able to extract the value because it's very likely a misconfiguration.